clusterloader2/pkg/config/template_provider.go

```diff
@@ -161,6 +161,19 @@ func (tp *TemplateProvider) TemplateInto(path string, mapping map[string]interfa
 	return decodeInto(b, obj)
 }
 
+// GetTestSuite creates test suite config from file specified by the given path.
+func GetTestSuite(path string) (api.TestSuite, error) {
```

Comment:
ReadTestSuite ?

or maybe LoadTestSuite is better

Reply:
Done + GetOverridesMapping -> LoadOverridesMapping to keep naming coherent.